File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/EosEkstra.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.ProRataBeregningTypeCti;

import java.io.Serializable;

public class EosEkstra implements Serializable {

    

    private ProRataBeregningTypeCti proRataBeregningType;

    private Integer redusertAntFppAr;

    private Double spt_eos;

    private Integer spt_pa_f92_eos;

    private Integer spt_pa_e91_eos;

    private Boolean vilkar3_17Aok;

}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/EosEkstra.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.kode.ProRataBeregningTypeCti
import java.io.Serializable

class EosEkstra : Serializable {
    var proRataBeregningType: ProRataBeregningTypeCti? = null
    var redusertAntFppAr: Int? = null
    var spt_eos: Double? = null
    var spt_pa_f92_eos: Int? = null
    var spt_pa_e91_eos: Int? = null
    var vilkar3_17Aok: Boolean? = null
}